src/getFiles.js

```diff
@@ -3,13 +3,10 @@ import minimatch from 'minimatch';
 import path from 'path';
 
 export default function getFiles(fromPath, exclude = []) {
-  const files = recursiveReadSync(fromPath)
-    .map(p => p.substr(fromPath.length + 1)) // get files relative to fromPath
+  return recursiveReadSync(fromPath)
     .filter(file =>
       exclude.every(excluded =>
-        !minimatch(file, path.join(excluded), { dot: true })
+        !minimatch(path.relative(fromPath, file), path.join(excluded), { dot: true })
```

Comment:
Can the `path.relative` call be cached outside of the function pasted to `exclude.every`?